README.md

```diff
@@ -5,61 +5,68 @@ GPU infrastructure and automation tools
 
 ## Overview
 
-The DeepOps project facilitates deployment of GPU servers and multi-node GPU clusters for Deep Learning and HPC environments, in an on-prem, optionally air-gapped datacenter or in the cloud.
+The DeepOps project encapsulates best practices in the deployment of GPU server clusters and sharing single powerful nodes (such as [NVIDIA DGX Systems](https://www.nvidia.com/en-us/data-center/dgx-systems/)). DeepOps can also be adapted or used in a modular fashion to match site-specific cluster needs. For example:
```

Comment:
It seems like we've removed all references to single-node/non-scheduler deployments. I feel like we need some mention somewhere that you can provision a single machine without a scheduler (this is how I use DeepOps in production every day) or use just some part of the project vs. an entire cluster deployment.

Reply:
👍

Reply:
Added...
```
* A single machine where no scheduler is desired, only NVIDIA drivers, Docker, and the NVIDIA Container Runtime
```